content/working_with/official_plugins/Orchestration/terraform.md

```diff
@@ -273,6 +273,50 @@ cloudify.nodes.terraform.Module:properties.resource_config
     * api_key: api key from Infracost, instructions how to retrieve it give in a [link](https://www.infracost.io/docs/)
     * enable: boolean, In order for it to work, must mark True.
 
+### OPA
+
+The `terraform.opa` interface operation evaluates an [Open Policy Agent (OPA)](https://www.openpolicyagent.org/) decision against a Terraform plan. Calling this interface operation will initialize Terraform (if it has not already been initialized), generate a Terraform plan, and then evaluate the decision against the provided OPA policies.
+
+The operation provides a thing wrapper around running `opa exec` against the Terraform plan in JSON format.
+
+OPA is configured by setting the desired parameters in `cloudify.nodes.terraform.Module:properties.opa_config`:
+
+* `installation_source` - Location to download the OPA binary from. Defaults to `https://github.com/open-policy-agent/opa/releases/download/v0.47.4/opa_linux_amd64_static`
+* `executable_path` - Location of an existing OPA binary on the system. If used, the binary will not be downloaded from the `installation_source` and the existing binary will be used for policy evaluations.
+* `config` - OPA configuration file to override any default OPA behavior.
+* `policy_bundles` - A list of one or more zipped policy [bundles](https://www.openpolicyagent.org/docs/latest/management-bundles/) for use in the policy evaluation. Policy budnles can be local to the blueprint archive, or they can be downloaded from remote sources. The format for policy bundles is described below.
```

Comment:
```suggestion
* `policy_bundles` - A list of one or more zipped policy [bundles](https://www.openpolicyagent.org/docs/latest/management-bundles/) for use in the policy evaluation. Policy bundles can be local to the blueprint archive, or they can be downloaded from remote sources. The format for policy bundles is described below.
```

Reply:
Is it possible also, to put the `policy_bundles` key indented beneath `config` in markdown?

Reply:
I'm not sure what you mean. In the example below, it's indented beneath `opa_config`. It's a key on `opa_config`, not `config`.